ement-room.el

```diff
@@ -2419,6 +2419,64 @@ these all require at least version 29 of Emacs):
                                ('remove "removed from"))
                              (ement--format-room space))))))
 
+;;;;; Admin/power commands
+
+(defun ement-room--kick-ban-interactive (prompt)
+  "Interactive arguments for kicking or un/banning a user.
+See `ement-room-kick-user', `ement-room-ban-user', `ement-room-unban-user'.
+Returns (user room session reason)."
+  (ement-room-with-highlighted-event-at (point)
```

Comment:
Taking a fresh look at this, it occurs to me that we need to allow the command to be run in any context, not just when looking at an undesirable message in a room buffer (i.e. as it is now, one would have to locate a message from a user in order to issue a kick/ban, and I can think of various circumstances in which one might want to just select a room and user and execute the action).  Of course, if point is on a message in a room buffer, we ought to offer the room and user as defaults.

Reply:
Agreed.  I had the same thought, but decided it could wait (I think the main difficulty at present is unbanning if you don't have an event from that user handy).  But yeah, this can certainly be improved.

Reply:
Done.